openfisca_core/json_to_test.py

```diff
@@ -0,0 +1,150 @@
+# -*- coding: utf-8 -*-
```

Comment:
You're right, this was confusing. 

I indeed chose to separate some logic specific to turning a JSON object into a test case in a separated flag to clear the `scenario` code.

This is renamed, tell me if it's ok for you.